test/integrate/proxywasm/proxywasm_wazero_test.go

```diff
@@ -0,0 +1,14 @@
+package wasm_test
```

Comment:
add build tag as same as in `test/integrate/proxywasm/proxywasm_wasmer_test.go`

Reply:
I think we agreed to leave this one enabled by default so that tests can run. so dismissing this